packages/perseus/src/widgets/numeric-input/numeric-input.testdata.ts

```diff
@@ -36,6 +37,305 @@ export const question1: PerseusRenderer = {
     },
 };
 
+// Used for Storybook.
+export const defaultQuestion: PerseusRenderer = {
+    content: "$5008 \\div 4 =$ [[\u2603 numeric-input 1]] ",
+    images: {},
+    widgets: {
+        "numeric-input 1": {
+            graded: true,
+            version: {
+                major: 0,
+                minor: 0,
+            },
```

Comment:
A lot of this stuff is optional (like `version` when we're on 0.0). It might make things easier to read if we just included the data that's important to the test.